lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -132,6 +133,11 @@ def failure_lines
               encoded_string(exception.message.to_s).split("\n").each do |line|
                 lines << "  #{line}"
               end
+              if @extra_failure_lines.is_a?(Array) && !@extra_failure_lines.empty?
```

Comment:
Doing the `@extra_failure_lines.is_a?(Array)` check silently ignores the `extra_failure_lines` option if the user didn't set it to an array.  I think I'd prefer to see this just assume it is an array and then in initialize, you can do:

``` ruby
@extra_failure_lines     = Array(example.metadata[:extra_failure_lines])
```

...which will ensure it is always an array (making it a blank array if the key is not provided).